From oliver@neukum.org Fri Oct 21 05:04:44 2005
From: Oliver Neukum <oliver@neukum.org>
To: "Guillaume GOURAT / NexVision" <guillaume.gourat@nexvision.fr>, greg@kroah.com
Subject: USB: Kaweth.c udelay patch
Date: Fri, 21 Oct 2005 14:01:35 +0200
Cc: zapman@interlan.net, stephane@u-picardie.fr, bhards@bigpond.net.au
Content-Disposition: inline
Message-Id: <200510211401.36195.oliver@neukum.org>

From: Guillaume GOURAT / guillaume.gourat@nexvision.fr

On some arch (like arm) udelay cannot be called with value greater that
2000.


Signed-off-by: Guillaume GOURAT / guillaume.gourat@nexvision.fr
Signed-off-by: Oliver Neukum <oliver@neukum.name>

---
 drivers/usb/net/kaweth.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- gregkh-2.6.orig/drivers/usb/net/kaweth.c
+++ gregkh-2.6/drivers/usb/net/kaweth.c
@@ -469,7 +469,7 @@ static int kaweth_reset(struct kaweth_de
 				0,
 				KAWETH_CONTROL_TIMEOUT);
 
-	udelay(10000);
+	mdelay(10);
 
 	kaweth_dbg("kaweth_reset() returns %d.",result);
 
